browser/blueprints.py

```diff
@@ -94,17 +96,17 @@ def action(project_id, action_type, frame):
     tracks = state.readable_tracks if info_changed else False
 
     if frames_changed:
-        img = state.get_frame(frame, raw=False)
-        raw = state.get_frame(frame, raw=True)
+        encode = lambda x: base64.encodebytes(x.read()).decode()
         edit_arr = state.get_array(frame)
+        img_payload = {'seg_arr': edit_arr.tolist()}
```

Comment:
This `seg_arr` is the array data for the annotations and does not need to be sent if `y_changed` is false.

Reply:
Nice, good call. I've moved the "seg_arr" key to inside the `if y_changed` block.